models/nexusmutual/ethereum/covers/nexusmutual_ethereum_product_information.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+    alias = 'product_information',
```

Comment:
please include `schema` in each model config block
we will eventually remove from project file